jbehave-support-core/src/main/java/org/jbehavesupport/core/rest/RestServiceSteps.java

```diff
@@ -80,6 +79,16 @@ public void verifyResponse(String application, String status, ExamplesTable data
         resolveHandler(application).verifyResponse(status, data);
     }
 
+    @Then("response from [$application] REST API is successful")
+    public void verifySuccessfulResponse(String application) {
+        resolveHandler(application).verifySuccessfulResponse(null);
```

Comment:
Just a thought - maybe we could provide overloaded variant of `verifySuccessfulResponse()` without any parameters? What do you think?

Reply:
What is the purpose of such method?

Reply:
To avoid `verifySuccessfulResponse(null)` so as to improve readability - my first thought when reading this code was that we are verifying null (instead of using some default verification) which to me looks weird - but maybe thats just me, I dont know :slightly_smiling_face:

Reply:
Oh, you mean the one on handler! I thought you want step method without parameter.
Well, we can, but it would be only wrapping the same call... Even I don't like having "magic parameters" in methods, same approach we already have in `sendRequest` and as it is only one parameter, it is reasonable readable to me.

Reply:
Up to you, I dont mind too much either way, just thought I would point it out :slightly_smiling_face: